.github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Polish Notebook(s)
+about: Finalize notebooks prior to publishing or promoting
+---
+
+## Related Issues and Dependencies
+
+…
+
+## The following checklist items have been completed for each notebook in this PR:
+
+- [ ] Proofread by other team member for grammar and spelling
+- [ ] Proofread by other team member for coherence and reproducibility (does the analysis make sense as you read through the code and markdown from top to bottom?)
```

Comment:
Just wanted to clarify, does "proofread for reproducibility" basically mean checklist item 9? Or is it something different?

Reply:
"proofread for reproduciblity" is more about, if you rerun a cell does the text around it still make sense or have the results changed in a way that causes the explanation and the results to be incongruous.

Item 9  "Assuming correct environment, clicking "run all" works without error" is just to check all the cells work and work in order.

Reply:
Gotcha, makes sense :+1: